### DISCUSSION TITLE
[Bug]:

### DISCUSSION THREAD

**Discussion by Prudhvi0717** [strimzi/strimzi-kafka-operator#8748] [Q&A] [answered]
### Bug Description

I have a setup with gke cluster 1.24 and strimzi 0.27.1 deployed and working fine from months.
Now i have created another dev cluster with kubernetes version 1.25 and deployed strimzi kafka cluster. But Operator is unable to deploy kafka due to following error:

Failure executing: POST at: https://10.84.0.1/apis/policy/v1beta1/namespaces/kafka/poddisruptionbudgets.

I have converted all my custom resources and crds using api-conversion-0.27.1 tool. (which i havent seen any major changes in crds as they already using v1beta2 written in them) and deployed again but still same error persists.

Why PDB's are being deployed with version v1beta1.. what should i do now?
@scholzj 

### Strimzi version

0.27.1

### Kubernetes version

1.25

### Installation method

Helm chart

### Infrastructure

GKE

**Comment by scholzj** [comment, marked as answer]
Strimzi 0.27 does not support Kubernetes 1.25 or newer. You should upgrade your Strimzi deployment. You can check the supported versions here: https://strimzi.io/downloads/
